packages/web-components/src/menu-item/menu-item.template.ts

```diff
@@ -14,69 +14,34 @@ const chevronRight16Filled = html.partial(
 
 export function menuItemTemplate<T extends MenuItem>(options: MenuItemOptions = {}): ElementViewTemplate<T> {
   return html<T>`
-  <template
-      aria-haspopup="${x => (x.hasSubmenu ? 'menu' : void 0)}"
+    <template
+      aria-haspopup="${x => (!!x.submenu ? 'menu' : void 0)}"
       aria-checked="${x => (x.role !== MenuItemRole.menuitem ? x.checked : void 0)}"
       aria-disabled="${x => x.disabled}"
-      aria-expanded="${x => x.expanded}"
+      aria-expanded="${x => (!!x.submenu ? 'false' : void 0)}"
       @keydown="${(x, c) => x.handleMenuItemKeyDown(c.event as KeyboardEvent)}"
       @click="${(x, c) => x.handleMenuItemClick(c.event as MouseEvent)}"
       @mouseover="${(x, c) => x.handleMouseOver(c.event as MouseEvent)}"
       @mouseout="${(x, c) => x.handleMouseOut(c.event as MouseEvent)}"
-  >
-          ${when(
-            x => x.role === MenuItemRole.menuitemcheckbox,
-            html<MenuItem>`
-              <div part="input-container" class="input-container">
-                <span part="checkbox" class="checkbox">
-                  <slot name="checkbox-indicator"> ${staticallyCompose(options.checkboxIndicator)} </slot>
-                </span>
-              </div>
-            `,
-          )}
-          ${when(
-            x => x.role === MenuItemRole.menuitemradio,
-            html<MenuItem>`
-              <div part="input-container" class="input-container">
-                <span part="radio" class="radio">
-                  <slot name="radio-indicator"> ${staticallyCompose(options.radioIndicator)} </slot>
-                </span>
-              </div>
-            `,
-          )}
+      @toggle="${(x, c) => x.toggleHandler(c.event as ToggleEvent)}"
+    >
+      <div part="indicator" class="indicator">
+        <slot name="indicator"> ${staticallyCompose(options.indicator)} </slot>
       </div>
```

Comment:
Question/thought - Can we make this and `submenu-glyph` just a slot and then add the corresponding classes to the default SVG's?

Reply:
Essentially all we'd have here is `<slot name="indicator"> ${staticallyCompose(options.indicator)} </slot>`

Reply:
When I tried this, it all fell apart because the grid really depended on the structure. BUT. I'll try again.

Reply:
I pulled this down and tried it - but I *might* have missed a configuration perhaps that we don't have in the storybook. :)

Reply:
@chrisdholt I think I got it working with flatter DOM, but just so I'm clear, this is what you were thinking? 

```js
const Checkmark16Filled = html.partial(
  `<svg class="indicator" fill="currentColor" ...`,
);
const chevronRight16Filled = html.partial(
  `<svg class="submenu-glyph" fill="currentColor" ...`,
);
```

Then would consumers would then need that `.indicator` or `.submenu-glyph` class on their slotted SVGs? 
Or should I also do `::slotted([slot="indicator"]), ::slotted([slot="submenu-glyph]])`  for styling and positioning slotted content?

Reply:
I think the selectors can stay as they are - the classes in the stylesheet would map to the default slotted content whereas the `::slotted()` selectors would be for anything coming in the light DOM.

Reply:
Okay! I think I got it all flattened. LMK what you think @chrisdholt

Reply:
Excellent!